### PR TITLE
Libraries: More parameter unit fixes

### PR DESCRIPTION
**Pull request by amilcarlucas**
Correct some units
Add some units
Correct some spelling errors

### CODE REVIEW

File: libraries/AP_Gripper/AP_Gripper.cpp [hunk 1]
```diff
@@ -31,20 +31,23 @@ const AP_Param::GroupInfo AP_Gripper::var_info[] = {
     // @Description: PWM value sent to Gripper to initiate grabbing the cargo
     // @User: Advanced
     // @Range: 1000 2000
+    // @Units: microseconds
```
Comment (OXINARF) [inline libraries/AP_Gripper/AP_Gripper.cpp]: Pretty sure we use pwm in these cases

File: libraries/AP_Stats/AP_Stats.cpp [hunk 2]
```diff
@@ -16,18 +16,21 @@ const AP_Param::GroupInfo AP_Stats::var_info[] = {
     // @Param: _FLTTIME
     // @DisplayName: Total FlightTime
     // @Description: Total FlightTime (seconds)
+    // @Units: s
```
Comment (OXINARF) [inline libraries/AP_Stats/AP_Stats.cpp]: This should be seconds

**Comment by rmackay9** [comment]
By the way, it's ok to have both an @Values and @BitMask in a single parameter's descriptions.  The GCSs seem to show the bitmask if they can (which is the better option).

**Comment by amilcarlucas** [comment]
Closing this, It moved to #5538 
@rmackay9 @Values seam to be wrong/outdated or incomplete, so I removed them.

**Pull request closed without merging**